<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BehaviorConfig_ShortenBase</class>
 <widget class="QWidget" name="BehaviorConfig_ShortenBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>156</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="kcfg_shortenOnPaste">
     <property name="text">
      <string>Shorten &amp;URLs on paste</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="kcfg_removeHttp">
     <property name="text">
      <string>Remove &quot;http://&quot; from shortened URLs</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Shortening ser&amp;vice:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>shortenPlugins</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="shortenPlugins">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="configPlugin">
       <property name="toolTip">
        <string>Configure plugin</string>
       </property>
       <property name="icon">
        <iconset theme="configure"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aboutPlugin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>About plugin</string>
       </property>
       <property name="icon">
        <iconset theme="help-about"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>kcfg_shortenOnPaste</tabstop>
  <tabstop>kcfg_removeHttp</tabstop>
  <tabstop>shortenPlugins</tabstop>
  <tabstop>configPlugin</tabstop>
  <tabstop>aboutPlugin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
